web/src/main/webapp/table.ts

```diff
@@ -216,14 +217,21 @@ export class TableView extends RemoteObject
         this.top.style.alignItems = "stretch";
 
         let menu = new TopMenu([
-            { text: "View", subMenu: new TopSubMenu([
-                { text: "Home", action: () => { TableView.goHome(this.page); } },
-                { text: "Refresh", action: () => { this.refresh(); } },
-                { text: "All rows", action: () => { this.showAllRows(); } },
-                { text: "No rows", action: () => { this.setOrder(new RecordOrder([])); } }
-            ])},
+            {
+                text: "View", subMenu: new TopSubMenu([
+                    { text: "Home", action: () => { TableView.goHome(this.page); } },
+                    { text: "Refresh", action: () => { this.refresh(); } },
+                    { text: "All columns", action: () => { this.showAllRows(); } },
+                    { text: "No columns", action: () => { this.setOrder(new RecordOrder([])); } }
+                ])
+            },
             {
                 text: "Combine", subMenu: combineMenu(this)
+            },
+            {
+                text: "Operation", subMenu: new TopSubMenu([
```

Comment:
This should probably move to the context menu and should apply to the selected columns.